pkgs/tools/security/ronin/default.nix

```diff
@@ -0,0 +1,25 @@
+{ pkgs, lib, bundlerApp }:
+
+bundlerApp {
+  pname = "ronin";
+  gemdir = ./.;
+  # ronin-rb requires ruby >= 3.0
+  ruby = pkgs.ruby_3_1;
+  exes = [
+    "ronin"
+    "ronin-db"
+    "ronin-exploits"
+    "ronin-fuzzer"
+    "ronin-payloads"
+    "ronin-repos"
+    "ronin-vulns"
+    "ronin-web"
+  ];
+
+  meta = with lib; {
+    description = "Ronin is a free and Open Source Ruby toolkit for security research and development";
```

Comment:
Should I replace this line with "Free and Open Source Ruby toolkit for security research and development"?

Reply:
Yes please. In general the `description` field should not include the package name.